src/runtime/particle-check.ts

```diff
@@ -65,16 +65,25 @@ export interface ProvideSlotConnectionSpecInterface extends ConsumeSlotConnectio
 }
 
 export class Check {
-  constructor(readonly target: CheckTarget, readonly expression: CheckExpression) {}
+  constructor(
+      readonly target: CheckTarget,
+      readonly fieldPath: string[],
+      readonly expression: CheckExpression) {}
 
   toManifestString() {
-    let targetString: string;
+    let targetString = this.targetString;
+    if (this.target.discriminator === 'CSCS') {
```

Comment:
What is `CSCS`?

Reply:
`ConsumeSlotConnectionSpec`. As opposed to `HandleConnectionSpec`.

There's two kinds of CheckTargets at the moment:

```
export type CheckTarget = HandleConnectionSpecInterface | ProvideSlotConnectionSpecInterface;
```

So this check is essentially saying "if it's a slot, do this".

Reply:
Can you add a comment?

Reply:
Done